src/sentry/static/sentry/app/views/organizationGroupDetails/utils.jsx

```diff
@@ -32,19 +32,43 @@ export async function fetchGroupEventAndMarkSeen(
 
   try {
     const data = await api.requestPromise(url, {query});
```

Comment:
could we call `fetchGroupEvent` here to reuse that code and not have to duplicate the `envNames` logic?